huaweicloud/services/as/resource_huaweicloud_as_group.go

```diff
@@ -768,66 +783,71 @@ func resourceASGroupUpdate(ctx context.Context, d *schema.ResourceData, meta int
 	return resourceASGroupRead(ctx, d, meta)
 }
 
+func forceDeleteASGroup(ctx context.Context, asClient *golangsdk.ServiceClient, d *schema.ResourceData) error {
+	if err := groups.ForceDelete(asClient, d.Id()).ExtractErr(); err != nil {
+		return fmt.Errorf("error deleting AS group %s: %s", d.Id(), err)
+	}
+
+	if err := checkASGroupRemoved(ctx, asClient, d.Id(), d.Timeout(schema.TimeoutDelete)); err != nil {
+		return fmt.Errorf("error deleting AS group %s: %s", d.Id(), err)
+	}
+	return nil
+}
+
+func deleteAllInstancesFromASGroup(ctx context.Context, asClient *golangsdk.ServiceClient, d *schema.ResourceData,
+	allIns []instances.Instance) error {
+	for _, ins := range allIns {
+		if ins.LifeCycleStatus != "INSERVICE" {
+			return fmt.Errorf("can't delete the AS group %s: some instances are not in INSERVICE but in %s, "+
```

Comment:
```suggestion
			return fmt.Errorf("cannot delete the AS group %s: some instances are not in 'INSERVICE' status, actually is '%s', "+
```
`can't` is the colloquial expression, `cannot` uses here be better. other places ditto.

Reply:
I will check the error message in another `PR`.